XamlControlsGallery/.editorconfig

```diff
@@ -0,0 +1,110 @@
+# EditorConfig  https://EditorConfig.org
+# Style as Code Minimal EditorConfig v.2020.02.06.0 https://styleascode.net
+
+# top-most EditorConfig file
+root = true
+
+# Use spaces for indentation.
+[*]
+indent_style = space
+
+# Code files
+[*.{cs,csx,xaml,xml}]
+charset = utf-8-bom
```

Comment:
Is there a specific reason for using BOM with UTF-8? The Unicode Standard is discouraging the use of BOM with UTF-8 so why not stick with simple UTF-8?

Reply:
I included -bom because it is present in the microsoft-ui-xaml repo's EditorConfig.

https://github.com/microsoft/microsoft-ui-xaml/blob/master/.editorconfig#L123

Reply:
I think changing it on the WinUI repro isn't worth the effort, but if you/we are creating a new file here, I don't think we should just inherit discouraged behaviors from other repos. 

But well that's just my opinion.

Reply:
I've removed it, seems like the better option.